webpack.config.js

```diff
@@ -23,7 +24,7 @@ const config = {
     ...getEntrypoints(),
   },
   output: {
-    filename: '[name].js',
+    filename: isDevelopment ? '[name].js' : '[name].[chunkhash].js',
```

Comment:
Can this be moved down to the [production](https://github.com/artsy/force/blob/ed61888c6c1c48f285f5054288cffa67c5cc3836/webpack.config.js#L130) branch? That way the config contains sensible defaults and all ENV related build config is isolated.

Reply:
Sure, I'll try that.

Reply:
Is chunkhash some new street drug?

Reply:
loll _Got some webpack?_